src/app/organisation/inviteusers/index.njk

```diff
@@ -30,77 +26,59 @@
     <form action="/organisation/inviteusers" autocomplete="off" class="form-group"
       method="post" role="form">
 
-      <h1 class="govuk-heading-xl">Invite a user</h1>
-      <p class="govuk-body-l">Please begin by providing the following information to register a user account in your organisation.</p>
-
-       <div class="govuk-form-group {{ m.form_group_class('invite', errors) }}">
+      <h1 class="govuk-heading-xl">{{__('title_invite_user')}}</h1>
+      <p class="govuk-body-l">{{__('invite_user_caption')}}</p>
 
-          <div class="govuk-form-group {{ m.form_group_class('fname', errors) }}">
```

Comment:
Someone from a background of ASP.net perhaps?

Removed the spans regarding field-validation, and removed the width:100% as they aren't needed.